torch/cuda/tunable.py

```diff
@@ -240,3 +256,73 @@ def read_file(filename: Optional[str] = None) -> bool:
+    assert tuning_is_enabled()
+
+    with open(filename) as file:
+        for line in file:
+            if line.startswith("Gemm"):
+                untuned_gemm = line.strip().split(",")[:]
+                [op_sig, data_type, layout] = untuned_gemm[0].split("_")
+
+                transA = True if layout[0] == "T" else False
+                transB = True if layout[1] == "T" else False
+
+                dtype = torch.half
+                if data_type == "float":
+                    dtype = torch.float32
+                elif data_type == "double":
+                    dtype = torch.float64
+                elif data_type == "BFloat16":
+                    dtype = torch.bfloat16
+                elif data_type == "Half":
+                    dtype = torch.half
+                elif data_type == "Float8_e4m3fn":
+                    dtype = torch.float8_e4m3fn
+                elif data_type == "Float8_e5m2":
+                    dtype = torch.float8_e5m2
+                elif data_type == "Float8_e4m3fnuz":
+                    dtype = torch.float8_e4m3fnuz
+                elif data_type == "Float8_e5m2fnuz":
+                    dtype = torch.float8_e5m2fnuz
+                elif data_type == "c10::complex<double>":
+                    dtype = torch.complex128
+                elif data_type == "c10::complex<float>":
+                    dtype = torch.complex64
+
+                if op_sig == "GemmTunableOp":
+                    [n, m, k] = [int(g) for g in untuned_gemm[1].split("_")[1:]]
+                    matA = (
+                        torch.rand(k, m, dtype=dtype, device="cuda").t()
+                        if transB
+                        else torch.rand(m, k, dtype=dtype, device="cuda")
+                    )
+                    matB = (
+                        torch.rand(n, k, dtype=dtype, device="cuda").t()
+                        if transA
+                        else torch.rand(k, n, dtype=dtype, device="cuda")
+                    )
+                    torch.mm(matA, matB)
+                elif op_sig == "GemmStridedBatchedTunableOp":
+                    [n, m, k] = [int(g) for g in untuned_gemm[1].split("_")[1:4]]
+                    [b] = [int(g) for g in untuned_gemm[1].split("_")[5:6]]
+                    matA = (
+                        torch.rand(b, k, m, dtype=dtype, device="cuda")
+                        if transB
+                        else torch.rand(b, m, k, dtype=dtype, device="cuda")
+                    )
+                    matB = (
+                        torch.rand(b, n, k, dtype=dtype, device="cuda")
+                        if transA
+                        else torch.rand(b, k, n, dtype=dtype, device="cuda")
+                    )
+                    matA = matA.transpose(1, 2) if transB else matA
+                    matB = matB.transpose(1, 2) if transA else matB
+                    torch.bmm(matA, matB)
+                else:
+                    print("error: unkown op")
```

Comment:
Please never use `print`, if this is a warning, how about `warnings:warn`
```suggestion
                    warnings.warn(f"error: unkown op {op_sig}")
```